src/app/game/game-draw/services/drawing.service.ts

```diff
@@ -47,7 +48,9 @@ export class DrawingService {
     });
   }
 
-  classify(answerInfo: FormData): Observable<any> { 
+  constructor(private http: HttpClient, private translationService: TranslationService) {}
```

Comment:
Double constructor? Another constructor in lines 42-49 which I guess is the right one. Getting an error for this

Reply:
Also error for duplicate identifier 'http' but I assume this will be fixed by removing duplicate constructor